explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+    ],
+    "title": "MyLampThing",
+    "@type": "saref:LightSwitch",
+    "securityDefinitions": {
+        "basic_sc": {"scheme": "basic", "in": "header"}
+    },
+    "security": "basic_sc",
+    "properties": {
+        "status": {
+            "@type": "saref:OnOffState",
+            "type": "string",
+            "forms": [{
+                "href": "https://192.168.2.134/status"
+            }]
+        }
+    },
+    "actions": {
+        "toggle": {
+            "@type": "saref:ToggleCommand",
+            "forms": [{
+                "href": "https://192.168.2.134/toggle"
+            }]
+        }
+    },
+    "events": {
+        "overheating": {
+            "data": {"type": "string"},
+            "forms": [{
+                "href": "https://192.168.2.134/oh"
+            }]
+        }
+    }
+}
+```
+### Thing Model definition
+One of the main intentions of a Thing Description is to provide a Consumer with all the details necessary to successfully interact with a Thing. In some IoT application scenarios, a fully detailed Thing Description, e.g., with communication metadata is not necessary (e.g., IoT ecosystems may implicitly handle communication separately), or may not be available because a new entity has not yet been deployed (e.g., IP address is not yet known). Sometimes, also a kind of class definition is required that forces capability definitions that should be available for all created instances (e.g., large-scale production of new devices).
+
+In order to address the above-mentioned scenarios or others, the Thing Model can be used that mainly provides the data model definitions within Things' Properties, Actions, and/or Events and can be potentially used as template for creating Thing Description instances. In the following a sample Thing Model is presented that can be seen as a model for the Thing Description instance in the first TD example above.
+
+```json
+{
+    "@context": ["https://www.w3.org/2022/wot/td/v1.1"], 
+    "@type" : "tm:ThingModel",
+    "title": "Lamp Thing Model",
+    "properties": {
+        "status": {
+            "description": "current status of the lamp (on|off)",
+            "type": "string",
+            "readOnly": true
+        }
+    },
+    "actions": {
+        "toggle": {
+            "description": "Turn the lamp on or off"
+        }
+    },
+    "events": {
+        "overheating": {
+            "description": "Lamp reaches a critical temperature (overheating)",
+            "data": {"type": "string"}
+        }
+    }
+}
+```
+Thing Model definitions are identified by the "@type": "tm:ThingModel". As the example shows, it does not provide details about a single Thing instance due to the lack of communication and security metadata. This specification presents a mechanism for deriving valid Thing Description instances from such Thing Model definitions. In addition, other design concepts are specified, including how to override, extend, and reuse existing Thing Model definitions. 
```

Comment:
using single code-quotes here instead of double quotes

Reply:
done